WalletWasabi.Fluent/ViewModels/AddWallet/Create/ConfirmRecoveryWordsViewModel.cs

```diff
@@ -50,8 +50,17 @@ public ConfirmRecoveryWordsViewModel(List<RecoveryWordViewModel> mnemonicWords,
 				.Bind(out _confirmationWords)
 				.Subscribe();
 
+#if DEBUG
+			var verificationWordsCount = 3;
+#else
+			var verificationWordsCount = 12;
+#endif
+
 			// Select random words to confirm.
-			confirmationWordsSourceList.AddRange(mnemonicWords.OrderBy(_ => new Random().NextDouble()).Take(3));
+			confirmationWordsSourceList.AddRange(
+				mnemonicWords
+						.OrderBy(_ => new Random()
+						.NextDouble()).Take(verificationWordsCount));
```

Comment:
The 12 words in random order!? Ummm isn't that too much?

Reply:
Uhm the randomness doesnt seem to work with this LINQ statement anyway, i'll take it out